samples/misc/vaultApprovedRelayer.py

```diff
@@ -0,0 +1,21 @@
+import balpy
+
+def main():
+    print("This test we'll set the user address as relayer and then remove it.")
+    print("Don't make sense setting the user as relayer, but it's just a test.")
+    print("==============================================")
+
+    bal = balpy.balpy.balpy("goerli")
+    userAddress = bal.web3.eth.default_account
+    print("Before ser approved relayer:", bal.balVaultHasApprovedRelayer(userAddress, userAddress))
+    print()
```

Comment:
if you want to skip a line you can also use something like `print(content, end='\n\n')`